java/server/src/org/openqa/selenium/grid/console/public/css/bootstrap.min.css

```diff
@@ -0,0 +1,12 @@
+/*!
```

Comment:
Should this file be saved or directly fetched over the internet? @AutomatedTester  what do you think?

Reply:
David said the website should work offline (in an intranet) that's why I've added it in the project.

Reply:
We want the site to work without having to fetch anything over the internet. The UI might be behind firewalls and might not have access to the internet